server/utils/matchUtils.ts

```diff
@@ -19,15 +19,21 @@ export const mapApiParamsForUi = (apiParams: BedSearchParameters): BedSearchPara
   maxDistanceMiles: apiParams.maxDistanceMiles.toString(),
 })
 
-export const mapApiCharacteristicForUi = (characteristic: CharacteristicPair) => {
-  const result = characteristic.name.startsWith('is') ? characteristic.name.slice(2) : characteristic.name
+export const mapApiCharacteristicForUi = (characteristic: string) => {
```

Comment:
This naming is a bit confusing, as we're not accepting an array and using `map` - how about `translateApiCharacteristicForUi`?

Reply:
Yeah that's fair